lib/rearchitecture/utils/widgets/dropdowns/zts_dropdown.dart

```diff
@@ -28,17 +28,22 @@ class _ZtsDropdownState extends State<ZtsDropdown> {
 
   @override
   Widget build(BuildContext context) {
-    final List<DropdownMenuEntry<Token>> entries = widget._availableTokens
-        .map(
-          (Token token) => DropdownMenuEntry<Token>(
-            label: token.name,
-            style: MenuItemButton.styleFrom(
-              foregroundColor: ColorUtils.getTokenColor(token.tokenStandard),
-            ),
-            value: token,
+    final List<DropdownMenuEntry<Token>> entries = widget._availableTokens.map(
+      (Token token) {
+        final String labelSuffix =
+            token.isCoin ? 'Coin' : token.tokenStandard.toString().short;
+
+        final String label = '${token.name} - $labelSuffix';
```

Comment:
The minus sign is a valid token name character. One could create a token with the name "ZNN-COIN" or "QSR-COIN". Therefore I think it's better to place the suffix between parentheses: `ZNN-COIN (zts1td...9qrkce)`